packages/react-tabs/src/components/Tab/useTab.ts

```diff
@@ -52,8 +52,9 @@ export const useTab_unstable = (props: TabProps, ref: React.Ref<HTMLElement>): T
     root: getNativeElementProps('button', {
       ref: useMergedRefs(ref, innerRef),
       role: 'tab',
+      'aria-disabled': disabled ? 'true' : undefined,
```

Comment:
if the root is a button, why are we using aria-disabled instead of simply disabled?

Reply:
I think this is because for composite widgets like tabs, comboboxes, menus, etc. we still let arrow key focus move through disabled items.

Reply:
The spread of props adds the disabled, so I don't need to add it directly.  Directly adding it was causing disabled="" in the non-disabled case .  The aria-disabled documentation has the case of applying it for a header button - which seems like the tab scenario.  It also mentions that it is "The aria-disabled attribute is similar to the HTML disabled attribute, but better. "

Reply:
so the spread is already adding 'disabled' to the button, so is aria-disabled better? I agree there is a question of focus on disabled tabs, but I don't feel that there is a case where we'd want to use disabled and aria-disabled at the same time